utils.ts

```diff
@@ -94,11 +172,16 @@ const filterOutDuplicateStops = (
   // added to the addendum field in Pelias. Therefore, there is still potential
   // for some transit stops without the "operator" tag to still be included in
   // search results.
+  // In HERE, transit stops will have one of the following categories.
+  const HERE_TRANSIT_STOP_CATEGORIES = ['Underground Train-Subway', 'Bus Stop']
   if (
     !feature.properties ||
     !feature.properties.addendum ||
-    !feature.properties.addendum.osm ||
-    !feature.properties.addendum.osm.operator
+    ((!feature.properties.addendum.osm ||
```

Comment:
oh boy.... this is starting to look a little wild. Maybe we can add some comments that are like this first part is for Pelias, the second part is for HERE

Reply:
there has gotta be a better way here... this is so hard for me to think about. All the negations and logical operators...